types/v2/types.benc.go

```diff
@@ -0,0 +1,258 @@
+// Code generated by bencgen golang. DO NOT EDIT.
```

Comment:
How do we regenerate this? Was trying to find some `go:generate`...

Reply:
Somehow the content of the types file didnt get included! Fixed.